specification/redis/resource-manager/Microsoft.Cache/stable/2017-10-01/redis.json

```diff
@@ -59,6 +59,90 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Cache/CheckNameAvailability": {
+      "patch": {
```

Comment:
This should be a POST instead of a PATCH

Reply:
Done